tests/Backend/Workspaces/Backend/SnowflakeWorkspaceBackend.php

```diff
@@ -71,7 +71,7 @@ public function dropTableColumn($table, $column)
     public function countRows($table)
     {
         $tables = $this->db->fetchAll(
-            sprintf("SHOW TABLES LIKE '%s' IN SCHEMA %", $table, $this->schema)
```

Comment:
Tohle vypadá podezřele, zkoumám jakoto že to fungovalo.

Reply:
Můj tip je, že `% == *`, takže to vytahlo tabulky ve všech schematech.

Reply:
Vygenerovalo to tohle `SHOW TABLES LIKE 'martin' IN SCHEMA ` A kupodivu s tím SNOWFLAKE nemá problém a znamená to to samé jako ``SHOW TABLES LIKE 'martin'`. A v tom co se tam testuje to vrací vždy credentials které mají právo jenom na jedno schéma takže to fungovalo validně.